oic.r.sensor/oic.r.sensor.json

```diff
@@ -0,0 +1,23 @@
+{
+  "id": "http://openinterconnect.org/iotdatamodels/schemas/oic.r.sensor.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description" : "Copyright (c) 2016 Open Connectivity Foundation, Inc. All rights reserved.",
+  "title": "Generic Sensor",
+  "definitions": {
+    "oic.r.sensor": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "boolean",
+          "description": "ReadOnly, true = sensed, false = not sensed."
```

Comment:
>ReadOnly [](start = 26, length = 8)

is there any way to enforce this in a field rather than text in the description?